src/ds/d500/d500-device.cpp

```diff
@@ -550,6 +557,21 @@ namespace librealsense
                                                                                DS5_ERROR_REPORTING,
                                                                                "Error reporting" );
 
+            if( _pid == ds::D555_PID )
```

Comment:
Isn't there a way to do this with some virtual function? Or add this to the `d555_device` ctor?

Reply:
Moved to d555_device constructor.